src/main/io/rcdevice_osd.c

```diff
@@ -0,0 +1,256 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "rcdevice.h"
+#include "rcdevice_osd.h"
+
+#include "common/maths.h"
+#include "common/time.h"
+#include "config/feature.h"
+#include "drivers/time.h"
+#include "drivers/vcd.h"
+#include "fc/config.h"
+#include "fc/rc_controls.h"
+#include "fc/runtime_config.h"
+#include "io/beeper.h"
+#include "io/osd.h"
+#include "sensors/gyroanalyse.h"
+
+#define VIDEO_BUFFER_CHARS_PAL 480
+
+static uint8_t columnCount = 30;
```

Comment:
This variable duplicates `displayPort->cols` and so should be removed. Just use `displayPort->cols` throughout.

Reply:
ok, i get it